stickler.yml

```diff
@@ -0,0 +1,1270 @@
+
```

Comment:
**[REQUIRED]**
Let's the right configurations to the `.stickler.yml` file. Lets remove the html mark up from this file.  
Check out the correct linter configurations for Ruby projects [here](https://github.com/microverseinc/linters-config/tree/master/ruby).

Reply:
That's the one we used and it's not working for us. Rubocop is allowed as it's described on the pathwright.

Reply:
Let's remember to name the file correctly. It should be `.stickler.yml` **not** `stickler.yml`